paasta_tools/cli/cmds/status.py

```diff
@@ -641,11 +641,11 @@ def format_marathon_task_table(tasks):
     return format_table(rows)
 
 
-def format_kubernetes_pod_table(pods):
+def format_kubernetes_pod_table(pods, verbose: int):
     rows = [("Pod ID", "Host deployed to", "Deployed at what localtime", "Health")]
     for pod in pods:
         local_deployed_datetime = datetime.fromtimestamp(pod.deployed_timestamp)
-        hostname = f"{pod.host}" if pod.host is not None else "Unknown"
+        hostname = f"{pod.host}" if pod.host is not None else PaastaColors.grey("N/A")
```

Comment:
I also changed this because `Unknown` implies we don't know where the pod is. We know where it is - it's nowhere.